content/docs/getting-started/setup.md

```diff
@@ -90,25 +69,8 @@ Here's a summary of how to log into cloud.gov. (See above for details.)
 
 Command line interface:
 
-{{% govcloud %}}
-
-Everyone: `cf login -a api.fr.cloud.gov --sso` 
-
-{{% /govcloud %}}
-
-{{% eastwest %}}
-GSA and EPA: `cf login -a api.cloud.gov --sso`
-
-Everyone else: `cf login -a api.cloud.gov`
-{{% /eastwest %}}
-
+Everyone: `cf login -a api.fr.cloud.gov --sso`
```

Comment:
We can consolidate this a bit:

![screen shot 2017-04-24 at 4 33 17 pm](https://cloud.githubusercontent.com/assets/391313/25362937/be52c14c-290b-11e7-866a-8f6a7e762708.png)